sync_merkleroots.go

```diff
@@ -0,0 +1,61 @@
+package walletclient
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/models"
+)
+
+// SyncMerkleRoots syncs merkleroots known to spv-wallet with the client database
+func (wc *WalletClient) SyncMerkleRoots(ctx context.Context, repo models.MerkleRootsRepository, timeoutMs time.Duration) error {
```

Comment:
IMHO we should remove timeout argument, and put in the documentation (godoc = comment above the method) that if somebody want to add timeout he need to pass context.WithTimeout().

Although the API will slightly differ from JS, but it actually already differ, because 
- we're passing a context which doesn't have representation in js
- timeout argument is required here and in js it is optional